File: test/dotnet/Trades.cs
using System;
using System.Collections.Generic;
using Mexc.Sdk;
using Newtonsoft.Json;
namespace dotnet
{
   public class Trades
   {
       public static void Main(string[] args)
       {
           var spot = new Spot();

           String symbol = "BTCUSDT";
           Dictionary<string, object> options = new Dictionary<string, object>();
           options.Add("limit", 500);



           var recentTradesRes = spot.Trades(symbol, options);
           Console.WriteLine("get recent trades");
           Console.WriteLine(JsonConvert.SerializeObject(recentTradesRes));
       }

   }
}

File: test/java/Trades.java
import Mexc.Sdk.*;

import java.util.HashMap;

public class Trades {
    public static void main(String[] args) {
        Spot client = new Spot();
        String symbol = "BTCUSDT";
        HashMap options = new HashMap<>();
        options.put("limit", 500);
        System.out.println("get recent trades " + client.trades(symbol, options));
    }
}